src/components/App.js

```diff
@@ -41,10 +53,10 @@ function App() {
             <UserContextProvider>
               <PhotosContextProvider>
                 <SearchContextProvider>
-                <NavDrawer darkMode={darkMode} setDarkMode={setDarkMode} />
+                <NavDrawer logOut={logOut} />
```

Comment:
I thought you liked Dark Mode! Why are you pulling it out?

Reply:
Please no. #saveDarkMode